kuttl-test.yaml

```diff
@@ -17,7 +17,6 @@ kind: TestSuite
 reportFormat: JSON
 reportName: kuttl-test-horizon
 timeout: 180
-namespace: openstack
```

Comment:
I think the pattern we have been using is like so [1] instead of removing the field completely.

[1] https://github.com/openstack-k8s-operators/keystone-operator/blob/fab7648d6c1074e9f82b6b21ba4b2a5310163f2e/kuttl-test.yaml#L20

Reply:
Hmm. We removed this in heat-operator instead of updating it. https://github.com/openstack-k8s-operators/heat-operator/commit/02940fab5059178a80e980963e42bad7920cd508 

Let me update this PR and heat-operator.

Note that, as far as I've tested, it seems NAMESPACE environment overrides the setting in `kuttl-test.yaml` so probably we should merge the change of install_yamls first.

Reply:
Okay, I will look at the `install-yamls` PR

Reply:
I had conversation about this with @cescgina , and I learned this namespace is not really used within jobs as we pass --namespace option as I'm adding here https://github.com/openstack-k8s-operators/install_yamls/pull/388/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R1406 .

We agreed that kuttl namespace as the default one we want to use, so the current PR should be correct.